components/brave_news/browser/resources/feed/Card.tsx

```diff
@@ -2,14 +2,31 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { color, font, radius, spacing } from '@brave/leo/tokens/css';
 import styled from "styled-components";
-import { color, effect, radius, spacing } from '@brave/leo/tokens/css'
+
+export const Header = styled.h2`
+  margin: 0;
+
+  font: ${font.primary.heading.h2};
+  color: ${color.text.primary};
+
+  --leo-icon-size: 18px;
+`
+
+export const Title = styled.h3`
+  margin: 0;
+
+  font: ${font.primary.default.regular};
+  color: ${color.text.primary};
+`
 
 export default styled.div`
-  background: ${color.container.background};
-  box-shadow: ${effect.elevation[7]};
-  border-radius: ${radius.m};
-  border: 1px solid ${color.gray[20]};
+  background: rgba(255, 255, 255, 0.3);
+  border-radius: ${radius.xl};
   color: ${color.text.primary};
-  padding: ${spacing.m};
+  padding: ${spacing["2Xl"]};
+
+  ${p => p.onClick && 'cursor: pointer'}
```

Comment:
This is a click handler on a div, so isn't going to be activatable via keyboard or screen reader. Can we find a way to give it a compatible role?

Reply:
Still faffing around with this - I've been doing a bit of reading on the best approach and it doesn't seem trivial (apparently just throwing everything in an `a` is frowned upon for accessibility, and doesn't play nice with out buttons anyway)